api/notd/api/api_v1.py

```diff
@@ -130,6 +131,11 @@ async def refresh_overlaps_for_all_collections_deferred():
         await notdManager.refresh_overlaps_for_all_collections_deferred()
         return RefreshCollectionOverlapsDeferredResponse()
 
+    @router.post('/collections/refresh-user-badges-deferred', response_model=RefreshAllUserBadgesDeferredResponse)
```

Comment:
this object should be RefreshUserBadgesForAllCollectionsResponse (and similar for request in the other file)